bitnami/grafana-operator/values.yaml

```diff
@@ -313,14 +326,12 @@ grafana:
   ## @param grafana.containerSecurityContext.enabled Enable containers security context
   ## @param grafana.containerSecurityContext.runAsUser User ID for the containers
   ## @param grafana.containerSecurityContext.runAsGroup Group ID for the containers
-  ## @param grafana.containerSecurityContext.fsGroup Filesystem Group ID for the containers
```

Comment:
Could you please elaborate on why you deleted the `fsGroup` ? It is used to configure the permissions in the volumes in some K8s clusters.

Reply:
because the kubernetes documentation says that for the individual containers `securityContext` the `fsGroup` attribute is not supported. -> 
* [podSecurityContext](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#podsecuritycontext-v1-core)
* [containerSecurityContext](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#securitycontext-v1-core)

That is also why it got removed from the `grafanas` CRD in the grafana-operator.

Reply:
yes, that's righ, I just found the issues please check the thread in the [original issue](https://github.com/bitnami/bitnami-docker-grafana/issues/17)

Reply:
https://github.com/grafana-operator/grafana-operator/issues/639